src/modules/media/components/DynamicActionMenu/DynamicActionMenu.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+	Button,
+	Dropdown,
+	DropdownButton,
+	DropdownContent,
+	MenuContent,
+} from '@meemoo/react-components';
+import clsx from 'clsx';
+import { FC, useRef, useState } from 'react';
+
+import { Icon, IconLightNames } from '@shared/components';
+import { useElementSize } from '@shared/hooks';
+
+import styles from './DynamicActionMenu.module.scss';
+import { ActionItem, DynamicActionMenuProps } from './DynamicActionMenu.types';
+
+const DynamicActionMenu: FC<DynamicActionMenuProps> = ({
+	className,
+	actions,
+	limit = 0,
+	onClickAction,
+}) => {
+	const listRef = useRef<HTMLUListElement>(null);
+	const size = useElementSize(listRef);
+
+	const totalWidth = 48 * actions.length + 8 * (actions.length - 1);
```

Comment:
Die arbitraire nummers hier mag je in een `DynamicActionMenu.const.ts` plaasten, ik vermoed dat de nummers voor het volgende dienen:
```
const DYNAMIC_ACTION_WIDTH = 48;
const DYNAMIC_ACTION_SPACER = 8;
// Dit is de 56 die gebruikt wordt maar geen idee of dat effectief de som is van de bovenste?
const DYNAMIC_ACTION_BOX = DYNAMIC_ACTION_WIDTH + DYNAMIC_ACTION_SPACER; 
```
Je mag die gerust beter noemen 😄

Reply:
Klopt, dat is inderdaad de combinatie van die spacers.